packages/ui/src/Notification.module.scss

```diff
@@ -0,0 +1,92 @@
+@use '../styles/constants' as c;
+
+.notification {
+  display: flex;
+  align-items: flex-start;
+  padding: c.$grid * 2 c.$grid * 3;
+  border-radius: c.$borderRadiusNotification;
+
+  @include c.typographyBodySmall;
+
+  .icon {
+    flex: 0 0 auto;
+    margin-right: c.$grid * 2;
+  }
+
+  .content {
+    display: flex;
+    flex: 1 1 auto;
+    align-items: center;
+    /*
+     * Note:
+     * The (min)height is equal to corresponding icon height. Necessary due to 'flex-start' alignment and
+     * specific alignment differences between single-line and multi-line text content.
+     */
+    min-height: c.$grid * 6;
```

Comment:
What happens if we remove the `min-height`?

Reply:
Content is not vertically aligned properly.

<img width="863" alt="Screenshot 2021-01-04 at 11 04 39" src="https://user-images.githubusercontent.com/5691407/103523780-ae2c9280-4e7c-11eb-9ebf-7efb13a33226.png">